spec/urc-signature-guide.md

```
@@ -0,0 +1,183 @@
+}
+```
+
+### Compatibility with the URC
+
+We can keep abi-encoding `Registration` and `Delegation` messages and continue to use `toMessagePoint()`. The output of `toMessagePoint()` is equivalent to an `object_root` that contains the `REGISTRATION_DOMAIN_SEPARATOR` or `DELEGATION_DOMAIN_SEPARATOR` already mixed in. What’s needed is to add additional code to the URC to create a `SigningData` from this `object_root` and mix-in the `COMMIT_BOOST_DOMAIN`. 
+
+Since `SigningData` is so simple, there’s no need for bespoke SSZ-encoding logic. The biggest lift will be for the “Delegation Module” (i.e., the module that handles signing registrations and delegations) to use ABI-encoding instead of SSZ when crafting the `object_root`.
+
+```solidity
+function _computeSigningRoot(bytes32 objectRoot) public pure returns (bytes32) {
+		return sha256(abi.encodePacked(objectRoot, COMMIT_BOOST_DOMAIN_SEPERATOR);
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)